### PR TITLE
chore: Add a .gitpod.yml file 

### PR DESCRIPTION
**Pull request by achals**
**What this PR does / why we need it**:
Trying out https://www.gitpod.io/ since it has allows a web-based development flow that easy for users to on board on to.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# This configuration file was automatically generated by Gitpod.
```
Comment (kevjumba) [inline .gitpod.yml:1]: should we remove this since we changed the gitpod file.

**Review by kevjumba — APPROVED** [review]

File: .gitpod.yml (hunk 1 above)
Comment (achals) [reply .gitpod.yml:1]: lol yea

Comment (achals) [reply .gitpod.yml]: updated

**Pull request merged**